packages/devtools-frontend-lynx/package.json

```diff
@@ -19,30 +19,30 @@
+    "auto-debug-e2etest": "autoninja -C out/Default && pnpm run debug-e2etest --",
+    "auto-debug-unittest": "DEBUG_TEST=1 pnpm run auto-unittest --",
+    "auto-e2etest": "autoninja -C out/Default && pnpm run e2etest --",
+    "auto-interactionstest": "autoninja -C out/Default && pnpm run interactionstest",
     "auto-unittest": "scripts/test/run_auto_unittests.py --no-text-coverage",
     "build": "autoninja -C out/Default",
     "build-release": "autoninja -C out/Release",
-    "check": "npm run check-json && npm run check-gn && npm run check-lint && npm run check-loc",
+    "check": "npm run check-json && pnpm run check-gn && pnpm run check-lint && pnpm run check-loc",
     "check-external-links": "third_party/node/node.py --output scripts/check_external_links.js",
     "check-gn": "third_party/node/node.py --output scripts/check_gn.js",
     "check-json": "third_party/node/node.py --output scripts/json_validator/validate_module_json.js",
-    "check-lint": "third_party/node/node.py --output scripts/test/run_lint_check_js.js && third_party/node/node.py --output scripts/test/run_lint_check_css.js && npm run check-lint-components-css",
+    "check-lint": "third_party/node/node.py --output scripts/test/run_lint_check_js.js && third_party/node/node.py --output scripts/test/run_lint_check_css.js && pnpm run check-lint-components-css",
     "check-lint-components-css": "third_party/node/node.py --output scripts/test/run_lint_check_css.js --glob 'front_end/**/*.ts' --syntax html",
-    "check-lint-css": "third_party/node/node.py --output scripts/test/run_lint_check_css.js && npm run check-lint-components-css",
+    "check-lint-css": "third_party/node/node.py --output scripts/test/run_lint_check_css.js && pnpm run check-lint-components-css",
     "collect-strings": "third_party/node/node.py --output third_party/i18n/collect-strings.js",
     "components-server": "third_party/node/node.py --output scripts/component_server/server.js",
-    "debug-e2etest": "DEBUG_TEST=1 npm run e2etest",
+    "debug-e2etest": "DEBUG_TEST=1 pnpm run e2etest",
     "debug-test": "third_party/node/node.py --output scripts/npm_test.js --debug-devtools",
-    "debug-unittest": "DEBUG_TEST=1 npm run unittest",
+    "debug-unittest": "DEBUG_TEST=1 pnpm run unittest",
     "e2etest": "third_party/node/node.py --output scripts/test/run_test_suite.js --config=test/e2e/test-runner-config.json",
     "eslint-test": "third_party/node/node.py --output node_modules/mocha/bin/mocha \"./scripts/eslint_rules/tests/*_test.js\"",
     "generate-dark-mode-styles": "third_party/node/node.py --output scripts/dark_mode/generate_dark_theme_sheet.js",
-    "generate-protocol-resources": "scripts/deps/generate_protocol_resources.py && git cl format --js",
-    "install-deps": "scripts/deps/manage_node_deps.py",
+    "generate-protocol-resources": "python ./scripts/deps/generate_protocol_resources.py && git cl format --js",
+    "install-deps": "python ./scripts/deps/manage_node_deps.py",
```

Comment:
There's a shebang in the top of these python file.

`#!/usr/bin/env vpython`

Therefore, we should not specify python commands explicitly.

https://chromium.googlesource.com/chromium/tools/depot_tools/master/vpython3

Maybe we should make sure that we can use vpython.

Reply:
On windows using pnpm to run python script you cant just use the .py file. It requires to know what to use. I could do same as gclient and wrap it in JS to run. Alternative have a windows command. I cant see what actually calls these though on a search of the repo